guides/17_Migration_to_3_0.md

```diff
@@ -159,13 +159,102 @@ handle them please remove.
+
+You can pass below `POSTLoggerOptions` to the `POSTLogger` constructor:
+```typescript
+interface POSTLoggerOptions {
+  url: string;
+  batchSize?: number;
+  headers?: Record<string, string>;
+  intervalMs?: number;
+  logLevel?: LogLevel;
+  metadata?: Record<string, string>;
+}
+```
+Please check `POSTLoggerOptions` documentation for more information.
+
+### Constructor
+
+```js
+// You need responses from server-side Chime API. See below for details.
+const meetingResponse = // The response from the CreateMeeting API action.
+const attendeeResponse = // The response from the CreateAttendee API action.
+
+// Before in 2.x
+const meetingSessionConfiguration = new MeetingSessionConfiguration(meetingResponse, attendeeResponse);
+const meetingSessionPOSTLogger = new MeetingSessionPOSTLogger(
+  'SDK',
+  configuration,
+  20, // LOGGER_BATCH_SIZE
+  2000, // LOGGER_INTERVAL_MS
+  URL,
+  LogLevel.INFO
+);
+
+// After in 3.x
+const options: POSTLoggerOptions = {
+  url: 'URL TO POST LOGS',
+  logLevel: LogLevel.INFO,
+  metadata: {
+    appName: 'SDK',
+    meetingId: meetingResponse.Meeting.MeetingId,
+    attendeeId: attendeeResponse.Attendee.AttendeeId,
+  },
+};
+const logger = new POSTLogger(options);
+```
+
+### Set `metadata` after `POSTLogger` creation
+
+This helps with Web application development using React frameworks or other UI frameworks where the component getting the server responses comes later based on an HTML form submission and not on initial load of the Web application.
+
+You may want to add logging in Providers, Hooks (if using React) in your application. Since, we have removed the `MeetingSessionConfiguration` dependency now, you can create `POSTLogger` earlier in the application flow and you can add `metadata` once you receive API responses from server upon a HTML form submission. This way, you can update `POSTLogger`'s `metadata` as per your application needs.
+
+```typescript
+const options: POSTLoggerOptions = {
+  url: 'URL TO POST LOGS',
+  logLevel: LogLevel.INFO,
+};
+// Create when your app loads.
+const logger = new POSTLogger(options);
+
+// User provides information to get the meeting and attendee responses.
+// Now, set the metadata on POSTLogger so that next logs can be identified and tied to a specific meetingId, attendeeId and an appName.
+const metadata: Record<string, string> = {
+  appName: 'SDK',
+  meetingId: meetingResponse.Meeting.MeetingId,
+  attendeeId: attendeeResponse.Attendee.AttendeeId,
+};
+
+// After this POSTLogger will include the metadata in HTTP POST request body when sending logs.
+logger.metadata = metadata;
+```
+
+### Set `headers` when creating `POSTLogger`
+
+You may want to add custom authentication `headers` with each HTTP POST request. You can provide such `headers` when constructing the `POSTLogger`.
+
+```typescript
+const options: POSTLoggerOptions = {
+  url: 'URL TO POST LOGS',
+  headers: {
+    Chime-Bearer: 'authentication-token',
```

Comment:
`Chime-Bearer` to `'Chime-Bearer'` (can't use `-` in an object key)

Reply:
Great catch! I tested this with meetingReadinessChecker and then realized but forgot to update here. Have updated now.